scripts/reap_stale_remote_branches.py

```
@@ -0,0 +1,75 @@
+        pr_out = run_cmd(f"gh pr list --repo {REPO} --head {branch} --state all --json number,state")
+        if pr_out:
+            try:
+                prs = json.loads(pr_out)
+                if prs:
+                    pr_state = prs[0].get("state")
+                    if pr_state in ["MERGED", "CLOSED"]:
+                        print(f"  [DELETE] Branch '{branch}' has {pr_state} PR #{prs[0]['number']}. Deleting from origin...")
+                        run_cmd(f"git push origin --delete {branch}")
+                        deleted_count += 1
+                        continue
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)